### PR TITLE
Remove broadcast command

### PR DESCRIPTION
**Pull request by marc0der**
This PR removes the `broadcast` command, the `flush broadcast` functionality, and all other broadcast-related code that is now obsolete.

### CODE REVIEW

File: src/main/bash/sdkman-availability.sh [hunk 1]
```diff
@@ -16,10 +16,10 @@
 #   limitations under the License.
 #
 
-function __sdkman_update_broadcast_and_service_availability() {
+function __sdkman_update_service_availability() {
 	local broadcast_live_id=$(__sdkman_determine_broadcast_id)
 	__sdkman_set_availability "$broadcast_live_id"
-	__sdkman_update_broadcast "$broadcast_live_id"
+	__sdkman_update_broadcast_id "$broadcast_live_id"
```
Comment (marc0der) [inline src/main/bash/sdkman-availability.sh:22]: This is still needed for the service availability check but will also go away in time. We use the hardcoded nginx `/broadcast/id` mapping here.

File: src/test/resources/features/checksum_verification.feature [hunk 2]
```diff
@@ -61,7 +61,8 @@ Feature: Verify checksums
 		And the candidate "grails" version "1.3.9" is installed
 		And the response headers file is created for candidate "grails" and version "1.3.9"
 		And the exit code is 0
-
+		
+	@manual
```
Comment (marc0der) [inline src/test/resources/features/checksum_verification.feature:65]: It seems to be impossible to build this locally. Am I the only one?

I'll be happy to bring this back if this is only occurring on my machine.

/cc @hgeraldino

**Review by helpermethod — APPROVED** [review]
LGTM!

File: src/test/resources/features/checksum_verification.feature (hunk 2 above)
Comment (hgeraldino) [reply src/test/resources/features/checksum_verification.feature:65]: I'll take a look

**Review by hgeraldino — APPROVED** [review]

File: src/test/resources/features/checksum_verification.feature (hunk 2 above)
Comment (marc0der) [reply src/test/resources/features/checksum_verification.feature:65]: Thank you!

**Pull request merged**